graphql-kotlin-schema-generator/build.gradle.kts

```diff
@@ -12,7 +12,7 @@ dependencies {
     api("com.graphql-java:graphql-java:$graphQLJavaVersion")
     api("org.jetbrains.kotlinx:kotlinx-coroutines-reactive:$kotlinCoroutinesVersion")
     api("com.fasterxml.jackson.module:jackson-module-kotlin:$jacksonVersion")
-    implementation(kotlin("reflect", kotlinVersion))
+    api(kotlin("reflect", kotlinVersion))
```

Comment:
I was getting the following errors locally without changing this

```
> Task :graphql-kotlin-federation:compileTestKotlin
w: Runtime JAR files in the classpath should have the same version. These files were found in the classpath:
    /Users/smyrick/.m2/repository/org/jetbrains/kotlin/kotlin-reflect/1.3.61/kotlin-reflect-1.3.61.jar (version 1.3)
    /Users/smyrick/.m2/repository/org/jetbrains/kotlin/kotlin-stdlib/1.4.0/kotlin-stdlib-1.4.0.jar (version 1.4)
    /Users/smyrick/.m2/repository/org/jetbrains/kotlin/kotlin-stdlib-common/1.4.0/kotlin-stdlib-common-1.4.0.jar (version 1.4)
w: Consider providing an explicit dependency on kotlin-reflect 1.4 to prevent strange errors
w: Some runtime JAR files in the classpath have an incompatible version. Consider removing them from the classpath
```